apps/vmq_generic_msg_store/README.md

```diff
@@ -0,0 +1,25 @@
+# Building VerneMQ with RocksDB support
+
+To build VerneMQ with RocksDB support (only), issue:
+
+```shell
+make with_rocksdb
+```
+
+To configure Verne to use RocksDB for the message store and the SWC store,
+add the following to your vernemq.conf file.
+
+```
+# Use RocksDB Message Store
+generic_message_store_engine = vmq_storage_engine_rocksdb
```

Comment:
I think `vmq_storage_engine_rocksdb` is meant as a module name.

We can probably still simplify this (maybe schema translations needed). The only requirement in Schema mappings is that a mapping is from an arbitrary term TO something prefixed with the actual app name (otherwise Cuttlefish wouldn't generate a correct app.config). The TO mapping is internal, of course.

Another clean solution would also be to keep the app name prefix in the names for the vernemq.conf (example: `vmq_generic_msg_store.backend = rocksdb`).